packages/@ember/application/instance.ts

```diff
@@ -324,51 +335,124 @@ ApplicationInstance.reopenClass({
   @namespace ApplicationInstance
   @public
 */
-class BootOptions {
-  constructor(options = {}) {
-    /**
-      Interactive mode: whether we need to set up event delegation and invoke
-      lifecycle callbacks on Components.
-
-      @property isInteractive
-      @type boolean
-      @default auto-detected
-      @private
-    */
-    this.isInteractive = Boolean(environment.hasDOM); // This default is overridable below
+class CBootOptions {
```

Comment:
Why the name? To differentiate from the exported `BootOptions` interface? Sometimes for things like this I'll do `_BootOptions` or similar instead to indicate that it's distinct and internal-only.